index.html

```diff
@@ -251,5 +251,4 @@ <h1 class="header-4">I'm always interested in hearing<br>about new projects, so
     </section>
     </div>
 </body>
-
-</html>
```

Comment:
- Please reduce the margin spacing between the `hero section bottom` to the top of the `My Recent Works` heading, as it might make those visiting your portfolio think there is not other content below the hero section. 

**Space** 

![image](https://github.com/NgachaIan/PersonalPortfolio/assets/58771507/40d1a45c-bd35-487b-8414-b5a7765aeb6c)